tools/donate-cpu-server.py

```diff
@@ -1447,6 +1466,7 @@ def server(server_address_port: int, packages: list, packageIndex: int, resultPa
                     s += '...'
                 print_ts('invalid command: "' + s + '"')
             connection.close()
+            continue
```

Comment:
The `continue` at the end of unnecessary but I added them consistently for readability.

Reply:
We could get rid of the `if elif` chain but I don't wanna start to refactor stuff as well.